File: https:/github.com/nishi-patidar/leetcode-tracker.git/4168-MirrorDistanceOfAnInteger/4168-MirrorDistanceOfAnInteger.java
// Last updated: 8/3/2026, 12:44:19 PM
class Solution {
    public int mirrorDistance(int n) {
        int original = n;
        int reversed = 0;
        int temp = n;

        while (temp > 0) {
            reversed = (reversed * 10) + (temp % 10);
            temp /= 10;
        }

        return Math.abs(original - reversed);
    }
}

File: https:/github.com/nishi-patidar/leetcode-tracker.git/4168-MirrorDistanceOfAnInteger/4168-MirrorDistanceOfAnInteger.cpp
// Last updated: 8/3/2026, 12:44:27 PM
#include <cmath>

class Solution {
public:
    int mirrorDistance(int n) {
        int original = n;
        int rev = 0;
        
        while (n > 0) {
            rev = rev * 10 + n % 10;
            n /= 10;
        }
        
        return std::abs(original - rev);
    }
};
